var/spack/repos/spack_repo/builtin/packages/pdi/package.py

```diff
@@ -23,6 +23,7 @@ class Pdi(CMakePackage):
     maintainers("jbigot")
 
     version("develop", branch="main", no_cache=True)
+    version("1.9.0", sha256="04fee7851c4f2a156daddf7eb2c3c3b0132d80d3f0e448cdeebda0b7c4595639")
```

Comment:
1.9.1 is out now, you should be able to copy https://github.com/pdidev/spack/blob/stable/packages/pdi/package.py#L39